settings/index.html

```diff
@@ -2,30 +2,36 @@
 <html>
   <head>
     <title>Job Tracker</title>
-    <script src="popup.js"></script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <script src="popup.js"></script>
     <script src="https://requirejs.org/docs/release/2.3.6/minified/require.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/2.0.3/fetch.js"></script>
   </head>
   <body>
-    <div class="container">
-        <div class="jumbotron" id="loggedin" style="display: none;">
-          <h3>Eztrackr</h3>
-          <p>Click the button to logout of Trello</p>
-          <button class="btn btn-primary" id="trello_helper_login">Logout</button>
-          <br>
-          <br>
-          <p><a href="https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=ZBR894KSWNJPS&currency_code=CAD" target="_blank" class="text-dark">Donate</a></p>
-        </div>
-        <div class="jumbotron" id="loggedout" style="display: none;">
-          <h3>Eztrackr</h3>
-          <p>Please click the button to authorize Trello so we can save the cards for you!</p>
-          <p>Once authorized, click on the extension again.</p>
-          <button class="btn btn-primary" id="trello_helper_logout">Authorize</button>
-          <br>
-          <br>
-          <p><a href="https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=ZBR894KSWNJPS&currency_code=CAD" target="_blank" class="text-dark">Donate</a></p>
-        </div>
+    <div id="container">
+        <header>
+          <h1><span class="red">Ez</span><span class="black">trackr</span></h1>
+        </header>
+        <main>
+          <section id="loggedin">
+            <p>Click the button to logout of Trello</p>
+            <button class="btn btn-primary" id="trello_helper_login">Logout</button>
+          </section>
+          <section id="loggedout">
+            <p>
+              <span>Please click the button to authorize Trello so we can save the cards for you!</span>
```

Comment:
Can you make this text normal size? It looks like the text is bigger than the standard size.